aws_lambda_builders/workflows/python_uv/exceptions.py

```diff
@@ -0,0 +1,32 @@
+"""
+Python UV specific workflow exceptions.
+"""
+
+from aws_lambda_builders.exceptions import LambdaBuilderError
+
+
+class MissingUvError(LambdaBuilderError):
+    """Exception raised when UV executable is not found."""
+
+    MESSAGE = (
+        "uv executable not found at {uv_path}. "
```

Comment:
It probably doesn't make sense to have a `uv_path` if the error is precisely that we can't find uv. (if there was a "uv path", then that would mean that we have `uv`)